pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -19,6 +19,10 @@
 # For the Python package set
 , packageOverrides ? (self: super: {})
 , buildPackages
+, pkgsBuildBuild
```

Comment:
In other places of Nixpkgs this will also need to be done eventually. Maybe its better having a single attribute that consists of the 6 `pkgs{theirHost}{theirTarget}` sets? That could be constructed already in `stage.nix`.